lib/ledger_sync/domains/operation/find.rb

```diff
@@ -25,7 +25,7 @@ def operate
         end
 
         def resource
-          @resource ||= resource_class.find_by(id: params[:id])
+          @resource ||= resource_class.find_by(id: params[:id], **params[:limit])
```

Comment:
To keep it consistent with the rest of the methods (I mean search), lets use `where` and instead of `limit`, use `query` params.
```suggestion
          @resource ||= resource_class.where(params[:query]).find_by(id: params[:id])
```